### PR TITLE
React to AuthN changes

### PR DESCRIPTION
**Pull request by HaoK**
@Praburaj 

Verified that manually external login works for Twitter

### CODE REVIEW

File: src/MusicStore/Controllers/AccountController.cs [hunk 1]
```diff
@@ -383,7 +383,7 @@ public ActionResult ExternalLogin(string provider, string returnUrl = null)
         [ValidateAntiForgeryToken]
         public async Task<ActionResult> ExternalLoginConfirmation(ExternalLoginConfirmationViewModel model, string returnUrl = null)
         {
-            if (User.Identity.IsAuthenticated)
+            if (User.IsLoggedIn())
```
Comment (Praburaj) [inline src/MusicStore/Controllers/AccountController.cs]: Should we call this method `IsSignedIn()` just to be consistent with `SignInManager`, `SignIn` and `SignOut` methods in various places?

Comment (HaoK) [inline src/MusicStore/Controllers/AccountController.cs]: Yeah probably, I'll switch to IsSignedIn

**Comment by Praburaj** [comment]
>Verified that manually external login works for Twitter

Other ones should be similar. But worth validating and running the tests once.

**Comment by HaoK** [comment]
I would expect they basically never ever pass on my machine :/

**Comment by HaoK** [comment]
Updated with renames, (ClaimsIdentity extensions were moved to IPrincipal)

**Comment by Praburaj** [comment]
:)

**Comment by Praburaj** [comment]
Looks good. :shipit:

**Comment by HaoK** [comment]
Updated with more renames

**Pull request closed without merging**